extensions/vertx/runtime/src/main/java/io/quarkus/vertx/core/runtime/context/SafeVertxContext.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.vertx.core.runtime.context;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import jakarta.enterprise.util.Nonbinding;
+import jakarta.interceptor.InterceptorBinding;
+
+/**
+ * Check that a method in invoke on a duplicated context and mark that context as safe.
+ * This interceptor is a companion on {@link VertxContextSafetyToggle}.
+ * It means that the user knows the context is not going to be access concurrently.
```

Comment:
I'm not sure what this actually means... but it should read "be access**ed** concurrently".

Reply:
Added a different sentence.